service/borrowerService.js

```diff
@@ -0,0 +1,73 @@
+let dao = require("../dao/borrowerDAO.js");
+let moment = require("moment");
+
+const winston = require("winston");
+const logger = winston.createLogger({
+  transports: [new winston.transports.Console()],
+});
+
+exports.returnBook = (loan, res) => {
+  dao.getLoan(loan, (err, result) => {
+    if (err) {
+      res.status(500).send("Internal Server Error");
+      logger.error(err);
+      return;
+    }
+
+    // Check if loan exists
+    if (result.length < 1) {
+      res.status(404).send("We could not find that loan in the database. ");
+      return;
+    } else if (result.length > 0 && result[0].dateIn !== null) {
+      // Check if the book is turned in
+      res.status(406).send("That book is already returned. ");
```

Comment:
400 would probably be the best status code to use here. 406 indicates that the server can't meet the requirement given by the Accept header of the request.